grapher/core/Grapher.tsx

```diff
@@ -457,7 +457,9 @@ export class Grapher
     @observable private selectedEntitiesInQueryParam: string[] = []
 
     @action.bound private setTimeFromTimeQueryParam(time: string) {
-        this.timelineHandleTimeBounds = getTimeDomainFromQueryString(time)
+        this.timelineHandleTimeBounds = getTimeDomainFromQueryString(time).map(
```

Comment:
Nice! this makes sense to me. The only thing I can think of going wrong is the order of operations (I can't yet wrap my head around when this.times is available :) ).

But the tests passing gives me confidence that times is available.

Reply:
Great, yeah I had the same concern which is why I did it as a PR. But from my testing it seems to look good.